tests/kuttl/test-suites/default/watcher/05-assert.yaml

```diff
@@ -9,3 +9,5 @@ commands:
       oc get secret -n watcher-kuttl-default --no-headers=true | [ $(grep -c ^watcher) == 0 ]
       oc get secret -n watcher-kuttl-default --no-headers=true | [ $(grep -c ^rabbitmq-transport-url-.*-watcher-transport) == 0 ]
       oc get transporturl -n watcher-kuttl-default --no-headers=true | [ $(grep -c ^watcher-kuttl-watcher-transport) == 0 ]
+      oc get keystoneservice -n watcher-kuttl-default --no-headers=true | [ $(grep -c ^watcher) == 0 ]
```

Comment:
doing oc get + grep is ok if we're checking for something that we don't know the exact name of and thus needs regex, but for resources that we know the name of we can also use an `error` file. An error file in kuttl works the same as an assert file but in reverse, if any of the resources listed there exist in the cluster the test will fail, here is a simple example from the keystone-operator https://github.com/openstack-k8s-operators/keystone-operator/blob/main/tests/kuttl/tests/keystone_scale/03-errors.yaml

Reply:
Thanks for the hint. I will move them to an error file when I have the name, then.